JavaScript2/Week4/homework.md

```diff
@@ -5,7 +5,7 @@
 ## Step 1: Lets get real comfortable with map, filter and sort.
 
 Use the carGenerator function to generate 10 cars. The file with the function is here: 
-[https://github.com/HackYourFuture-CPH/JavaScript/blob/master/Week4/code/carGenerator.js](/Week4/code/carGenerator.js)
+[https://github.com/HackYourFuture-CPH/JavaScript/blob/organizedJSModules/JavaScript2/Week4/code/carGenerator.js](https://github.com/HackYourFuture-CPH/JavaScript/blob/organizedJSModules/JavaScript2/Week4/code/carGenerator.js)
```

Comment:
Is it correct that it links to a branch? 
If you link relative you don't need to include the whole path.